src/index.html

```diff
@@ -10,7 +10,58 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    action="/index.html"
+    method="post"
+    data-qa="big"
+    class="search-form"
+  >
+    <label
+      for="bigSearch"
+      class="search-label"
+    >
+      <img
+        src="images/Search.svg"
+        alt="search icon"
+        class="search-icon"
+      >
+    </label>
+    <input
+      type="search"
+      name="bigSearch"
+      id="bigSearch"
+      class="big search"
```

Comment:
following the BEM naming convention it should be rather `class=search search--big` just use the modifier class to modify element styling